_posts/2020-05-07-health-groups.adoc

```diff
@@ -0,0 +1,235 @@
+= Health Groups - experimental MicroProfile Health extension in SmallRye
+
+If you are reading this blog, you have probably already heard about
+https://github.com/eclipse/microprofile-health[Eclipse MicroProfile Health]
+technology. In this article, we will introduce a new experimental extension to
+MicroProfile Health 2.2 called Health groups.
+
+All of the source code that we will present in this article is available at this
+GitHub repository https://github.com/xstefank/health-groups.
+
+== MicroProfile 2.2 health checks
+
+As you know, to define a health check procedure in MicroProfile Health you need
+something like this:
+
+[source,java]
+----
+package io.xstefank;
+
+import org.eclipse.microprofile.health.HealthCheck;
+import org.eclipse.microprofile.health.HealthCheckResponse;
+import org.eclipse.microprofile.health.Readiness;
+
+import javax.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped // 1. CDI scope
+@Readiness         // 2. Qualifier (Readiness, Liveness, Health)
+public class ReadinessHealthCheck implements HealthCheck {   // 3. implements HealthCheck interface
+    
+    @Override
+    public HealthCheckResponse call() {
+        return HealthCheckResponse.up(ReadinessHealthCheck.class.getSimpleName());
+    }
+}
+----
+
+So just to recap, to define a health check your class:
+
+- needs to be a CDI bean
+- needs to have one of the health qualifiers (Readiness, Liveness, or deprecated Health)
+- must implement the HealthCheck interface
+
+This is everything that you can use in MicroProfile 2.2. But what if you have a
+use case that doesn't fall into one of Readiness or Liveness categories? The answer
+is custom health groups.
+
+== Health Groups
+
+Quarkus 1.4.0 (with SmallRye Health 2.2.0) comes with a new addition to this API 
```

Comment:
I'd personally put it the other way, as this is a SmallRye blog and the article is focused on SmallRye Health: "SmallRye Health 2.2.0, which is already present in Quarkus 1.4, comes with ..."